library/weno/src/AdminProperties.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * AdminProperties class.
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Rx\Weno;
+
+class AdminProperties
+{
+
+    public function __construct()
+    {}
+
+
+
+    public function addNarcotics()
+    {
+
+  /*
+  * Import the narcotics into the table database from SQL file
+  *
+  */
+        $sqlNarc = file_get_contents('../../contrib/weno/narc.sql');
+
+        sqlInsert($sqlNarc);
```

Comment:
how long does this import take?
If it takes awhile, let me know, and I can show you how to drastically reduce the time.

Reply:
That statement was from the first round before switching to the SQL version. I don't know how long it takes now.

Reply:
Are you saying this function is no longer used? If so, then would remove it.

Reply:
No, I am saying the words "Be patient this can take a while" can be removed.

Reply:
Use the sqlStatementNoLog() function here instead.